src/app/forms/components/approval-type/approval-type.component.ts

```diff
@@ -0,0 +1,195 @@
+import { AfterContentInit, ChangeDetectorRef, Component, Injector, Input, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { AbstractControlDirective, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { BehaviorSubject, Observable, Subscription, combineLatest } from 'rxjs';
+import { BaseControlComponent } from '../base-control/base-control.component';
+import { CustomFormGroup, FormNodeEditTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
+
+type Segments = {
+  techRecord_approvalTypeNumber1: Observable<string | undefined>;
+  techRecord_approvalTypeNumber2: Observable<string | undefined>;
+  techRecord_approvalTypeNumber3: Observable<string | undefined>;
+  techRecord_approvalTypeNumber4?: Observable<string | undefined>;
+};
+
+@Component({
+  selector: './app-approval-type-input',
+  templateUrl: './approval-type.component.html',
+  styleUrls: ['./approval-type.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: ApprovalTypeInput,
+      multi: true
+    }
+  ]
+})
+export class ApprovalTypeInput extends BaseControlComponent implements OnInit, OnDestroy, AfterContentInit {
+  @Input() isEditing = false;
+  @Input() approvalType?: string;
+
+  public form!: CustomFormGroup;
+
+  private techRecord_approvalTypeNumber1_: BehaviorSubject<string | undefined> = new BehaviorSubject<string | undefined>(undefined);
+  private techRecord_approvalTypeNumber2_: BehaviorSubject<string | undefined> = new BehaviorSubject<string | undefined>(undefined);
+  private techRecord_approvalTypeNumber3_: BehaviorSubject<string | undefined> = new BehaviorSubject<string | undefined>(undefined);
+  private techRecord_approvalTypeNumber4_: BehaviorSubject<string | undefined> = new BehaviorSubject<string | undefined>(undefined);
+  private techRecord_approvalTypeNumber1$: Observable<string | undefined>;
```

Comment:
I'd love a different variable name but I'm not sure what? This kind of implies that there are 4 numbers, when really it's 4 parts of 1 number

Reply:
Agreed, how about `techRecord_approvalTypeNumberPartx`

Reply:
Not sure we even need the techRecord? These values are not on a techRecord

Reply:
approvalTypeNumberPartx sounds good to me @naathanbrown  @andrewf-bjss